static/js/tools/scatter/info.tsx

```diff
@@ -42,7 +42,7 @@ function Info(): JSX.Element {
       <ul>
         <li>
           <b>
-            Prevalence Of Coronary Heart Disease vs Max Temperature Relative To
+            Prevalence Of Coronary Heart Disease vs. Max Temperature Relative To
             2006 RCP 4.5 Difference Relative To Base Date
```

Comment:
Below we call it "Projected temperature rise", should we call it the same?

Reply:
Ditto in Map

Reply:
![image](https://user-images.githubusercontent.com/4375037/163254533-03563745-eafd-4607-be98-bb31ee2251d7.png)

Reply:
another nit: s/Non-Biogenic Emission Source, Carbon Dioxide/Non-Biogenic Carbon Dioxide/ ?

Reply:
thanks for the catch - updated